src/main/menus/tray.js

```diff
@@ -29,6 +29,14 @@ function createTemplate(mainWindow, config) {
     }), {
       type: 'separator'
     }, {
+      label: 'Settings',
+      click: () => {
+        mainWindow.loadURL('file://' + __dirname + '/browser/settings.html');
+        mainWindow.show();
```

Comment:
@yuya-oc  'mainWindow.webContents.send('switch-tab', i);'? Why would that be needed for settings?